Doc/library/os.rst

```diff
@@ -3396,6 +3396,9 @@ written in Python, such as a mail server's external command delivery program.
    The positional-only arguments *path*, *args*, and *env* are similar to
    :func:`execve`.
 
+   The `path` parameter in `posix_spawn()` is a pathname that identifies
+   the new process image file to execute.
```

Comment:
In the Linux manpage, it's called "the executable file" rather than "the new process image file to execute":

```
 With  posix_spawn(),  the  executable file is specified as a
       pathname (which can be absolute or relative).
```

subprocess doc uses "The executable argument specifies a replacement program to execute." to document the **executable** argument.

I propose (I let you reformat it):

```
The *path* parameter is the path to the executable file.
The *path* should contain a directory.
Use :func:`posix_spawnp` to pass an executable file without directory.
```